.github/workflows/stale.yml

```diff
@@ -7,20 +7,23 @@ on:
 
 jobs:
   stale:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/stale@v3.0.7
+    - uses: actions/stale@v3
```

Comment:
Is there a reason for upgrading the version of the dependencies? Will it break anything? Maybe write that in your commit message too?

Reply:
It is to remain forward-compatible. As per semver, changes in the minor and patch versions will not cause breaking changes. I have also modified the commit message accordingly.